server/rest/src/test/java/org/infinispan/rest/resources/ContainerResourceTest.java

```diff
@@ -343,13 +374,22 @@ public void testConfigListener() throws InterruptedException, IOException {
          AssertJUnit.assertTrue(sseListener.openLatch.await(10, TimeUnit.SECONDS));
 
          // Assert that all of the existing caches and templates have a corresponding event
-         sseListener.expectEvent("create-template", TEMPLATE_CONFIG);
```

Comment:
this test fails randomly because the order is not always the same @tristantarrant